docs/source/tutorials/introductory/logging-first-model/step1-tracking-server.rst

```diff
@@ -0,0 +1,66 @@
+Starting a Tracking Server with MLflow
+======================================
+
+Before diving into MLflow's rich features, let's set up the foundational components: the MLflow
+Tracking Server and the MLflow UI. This guide will walk you through the steps to get both up and running.
+
+Setting Up MLflow
+-----------------
+
+The first thing that we need to do is to get MLflow.
+
+Step 1: Install MLflow from PyPI
+--------------------------------
+
+MLflow is conveniently available on PyPI. Installing it is as simple as running a pip command.
+
+.. code-section::
+    .. code-block:: bash
+        :caption: Install the Latest MLflow Version from PyPI
```

Comment:
Potentially controversial but I think all captions strongly decrease readability. IMO we should change caption rendering to the gif "italics under the block" style or remove.

Reply:
not controversial AT ALL. Removed them all :D They were super distracting!